Bixal/move.mil
# Add email phone types

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- Changes the email field from email to text
- Add a one-time script to fix the weight scales data. They are not part of the xml, and the fields changes affected their data.
- Adds email type and add it to the email field
- Adds voice field to phone paragraph
- Changes the locations controller response to include the types
- Adds the new fields in the front-end just as a reference (also so it doesn't break)

## Testing

To verify the changes proposed in this pull request…

1. pull changes, config:import, cr, all that
1. Upload xml file
1. run drupal mails (to keep the weight scales data right since we aren't touching weight scales data)
1. go to locator maps, and look for a location, it should look like this:

## Screenshots

![screen shot 2019-02-21 at 2 24 34 pm](https://user-images.githubusercontent.com/37001835/53196163-53ba1e00-35e5-11e9-9e7a-ac61ab47dbe2.png)
